drivers/coindcx.js

```diff
@@ -0,0 +1,53 @@
+const Driver = require('../models/driver');
+const request = require('../lib/request');
+const Ticker = require('../models/ticker');
+const { parseToFloat } = require('../lib/utils.js');
+
+/**
+ * @memberof Driver
+ * @augments Driver
+ */
+class Coindcx extends Driver {
+  /**
+   * @augments Driver.fetchTickers
+   * @returns {Promise.Array<Ticker>} Returns a promise of an array with tickers.
+   */
+  async fetchTickers() {
+    const symbols = await request('https://api.coindcx.com/exchange/v1/markets_details');
+    const pairs = [];
+
+    symbols.forEach((el) => {
+      pairs[el.symbol] = {
+        quote: el.base_currency_short_name,
+        base: el.target_currency_short_name,
+        quoteName: el.base_currency_name,
+        baseName: el.target_currency_name,
+      };
```

Comment:
Hey! These are reversed. If I look at the BTCINR market; the close they provide is the Bitcoin price in Rupees!

Reply:
@nickpater oh, true! Updated.

Reply:
Oh noes, I think I made a mistake. `base_currency_short_name` is INR, mh you had it correct the first time :sweat_smile: